pkg/reconciler/reconciler.go

```diff
@@ -97,7 +97,13 @@ func NewBase(opt Options, controllerAgentName string, images pipeline.Images) *B
 	if recorder == nil {
 		// Create event broadcaster
 		logger.Debug("Creating event broadcaster")
-		eventBroadcaster := record.NewBroadcaster()
+
+		correlatorOptions := record.CorrelatorOptions{
+			// The default burst size is 25
+			BurstSize: 50,
+			QPS:       1,
```

Comment:
Is this the number of queries per second that we can emit? I couldn't figure out after reading the godoc for this

Reply:
Yes, the way I understood this is the QPS we allow after a burst